packages/core/src/AvatarGroup/AvatarGroup.tsx

```diff
@@ -59,19 +63,36 @@ const getSpacingValue = (
   }
 };
 
+const getFontSize = (size: HvAvatarGroupProps["size"]) => {
+  switch (size) {
+    case "xs":
+      return "1em";
+    case "sm":
+      return "1.25em";
+    case "md":
+      return "1.5em";
+    case "lg":
+      return "1.75em";
+    case "xl":
+      return "3em";
+    default:
+      return "1em";
+  }
+};
+
 /**
  * The AvatarGroup component is used to group multiple avatars.
  */
 export const HvAvatarGroup = forwardRef<HTMLDivElement, HvAvatarGroupProps>(
   (props, ref) => {
     const {
       className,
+      style,
       classes: classesProp,
       children,
       size = "sm",
       spacing = "loose",
       direction = "row",
-      toBack = true,
```

Comment:
This deprecation looks like a breaking change since we are allowing users to use `toBack` but the prop doesn't do anything. Shouldn't we deprecate it, telling the users that this will be removed in the next major, but the prop still work?